internal/codegen/stencil_test.go

```diff
@@ -121,3 +122,25 @@ func ExampleStencil_PostRun() {
 	// Output:
 	// hello
 }
+
+func TestStencilPostRunError(t *testing.T) {
+	var (
+		name    = "TestStencilPostRunError"
+		command = strings.ToLower("invalidPostRunCommand")
```

Comment:
question: why does this need to be all lowercase?

If it does need to be lowercase, why not just hardcode it that way and save the function call?

Reply:
it's for cosmetic purpose, commands usually come in lowercase, but `invalidpostruncommand` looks like a typo. Do you recommend `command = "invalidPostRunCommand"` or `command = "invalidpostruncommand" instead?

Reply:
It doesn't matter since it's a test value. Just use the first option.